bundles/binding/org.openhab.binding.zwave/database/ge/ZW3101.xml

```diff
@@ -68,9 +64,11 @@
 			<Maximum>255</Maximum>
 			<Size>1</Size>
 			<Label lang="en">On/Off Command Dim Rate</Label>
-			<Help lang="en">This value indicates in 10 millisecond resolution, how often the dim level will change. For example, if you set this parameter to 1, then every 10ms the dim level will change. If you set it to 255, then every 2.55 seconds the dim level will change.</Help>
+			<Help lang="en"><![CDATA[This value indicates in 10 millisecond resolution, <br>
```

Comment:
Same comment here that I made previously - breaks shouldn't be added like this - only to enforce an end of line or new paragraph...